stable/phpmyadmin/templates/_helpers.tpl

```diff
@@ -45,6 +45,6 @@ Return the proper image name
 {{- define "phpmyadmin.image" -}}
 {{- $registryName :=  .Values.image.registry -}}
 {{- $repositoryName := .Values.image.repository -}}
-{{- $tag := .Values.image.tag -}}
+{{- $tag := .Values.image.tag | quote | trimPrefix "\"" | trimSuffix "\"" -}}
```

Comment:
You can use just `trim` which removes space from either side of a string.

Reply:
It doesn't work as expected, I checked it with @tompizmor and we obtain a weird error using
```
{{- $tag := .Values.image.tag | quote | trim "\"" -}}
```
the error is
```
executing "phpmyadmin.image" at <trim>: wrong number of args for trim: want 1 got 1
```

Reply:
Should we use *trimAll* instead? https://github.com/Masterminds/sprig/blob/master/docs/strings.md#trimall